### ISSUE TITLE
StagingFailed error for pushing .net application windows stack

### ISSUE THREAD

**Issue by cfmagnum** [cloudfoundry/diego-windows-release#24]
CloudFoundary version-250
diego/1.4.1
windows windows2012R2
Cloud- aws 
windows cell size-r3.xlarge

Hi i am trying to push a .net application on windows stack.But after starting the applicationn its giving staging failed error.
I am pushing app from repo  https://github.com/cloudfoundry/wats

Using Buildpack is  https://github.com/ryandotsmith/null-buildpack

Command for pushing app are as followed.

**cf push windows-test-app-1 -s windows2012R2 -b https://github.com/ryandotsmith/null-buildpack.git -m 1G --no-start**
**cf start windows-test-app-1**
I am attaching output of  CF_TRACE 
[cf_trace=true.txt](https://github.com/cloudfoundry/diego-release/files/823559/cf_trace.true.txt)
.

on checking logs with command 
**cf logs windows-test-app-1 --recent**
![screen shot 03-07-17 at 12 09 pm](https://cloud.githubusercontent.com/assets/13657881/23644759/ed2694f6-032e-11e7-9116-163858a9b35b.PNG)

Can you please assist us with the problem?
Thanks in Advance

**Comment by sunjayBhatia** [comment]
@cfmagnum can you confirm that when you run:

```
cf push windows-test-app-1 -s windows2012R2 -b https://github.com/ryandotsmith/null-buildpack.git -m 1G --no-start -p ./NoraPublished
cf start windows-test-app-1
```

From inside `<PATHTOWATS>/wats/assets/nora`

That the app still cannot stage?

**Comment by cfmagnum** [comment]
Yes that apps still don't stage.

Is there is any compatibility issue of diego version with windows cell?

**Comment by sunjayBhatia** [comment]
@cfmagnum can you provide log output for pushing with the commands I provided above and the version of DiegoWindows/GardenWindows

**Comment by cfmagnum** [comment]
I have already attached a CF_TRACE.txt while starting the app.

**Comment by cfmagnum** [comment]
sorry for above comment

for 1st command logs are
[cf_While_push.txt](https://github.com/cloudfoundry/diego-windows-release/files/824758/cf_While_push.txt)
for 2nd command
[cf_While_start.txt](https://github.com/cloudfoundry/diego-windows-release/files/824760/cf_While_start.txt)

and recent logs are same

![screen shot 03-07-17 at 08 26 pm](https://cloud.githubusercontent.com/assets/13657881/23662061/6dd2992a-0374-11e7-8478-eb4d2da5450a.PNG)

**Comment by natalieparellano** [comment]
@cfmagnum can you provide the versions of [diego-windows-release](https://github.com/cloudfoundry/diego-windows-release/releases) and [garden-windows-release](https://github.com/cloudfoundry/garden-windows-release/releases) that you are using?

**Comment by cfmagnum** [comment]
Hi,
We are using the following version.

https://github.com/cloudfoundry/diego-windows-release/releases/download/v0.467/generate.exe
 https://github.com/cloudfoundry/diego-windows-release/releases/download/v0.467/DiegoWindows.msi
 https://github.com/cloudfoundry/garden-windows-release/releases/download/v0.177/GardenWindows.msi
 https://github.com/cloudfoundry/diego-windows-release/releases/download/v0.467/hakim.exe
 https://github.com/cloudfoundry/garden-windows-release/releases/download/v0.177/setup.ps1

Regards 
Ishwar

**Comment by cfmagnum** [comment]
We did RDP on window cell we are able to see some error in event viewer.

{"timestamp":1488899836.165504456,"process_id":4852,"source":"metron","log_level":"warn","message":"Failed to connect to etcd. Number of tries: 1. Backing off for 932Âµs.","data":{"error":"sync cluster failed"},"file":"C:/diego-windows-release/loggregator/src/metron/backoff/backoff.go","line":33,"method":"metron/backoff.Connect"


IS that issue related with this.

**Comment by frodenas** [comment]
The staging is failing due to https://github.com/cloudfoundry/garden-windows/issues/15

**Comment by sunjayBhatia** [comment]
@cfmagnum We're you able to work this out? We solved #27 and it looks like you've gotten much farther than the issues you were having here

**Comment by sunjayBhatia** [comment]
closing due to inactivity, we can open this again if need be @cfmagnum
